hack/jenkins/cron/cleanup_and_reboot_Linux.sh

```diff
@@ -36,11 +36,77 @@ logger "cleanup_and_reboot is happening!"
 # kill jenkins to avoid an incoming request
 killall java
 
+# clean minikube left overs
+echo -e "\ncleanup minikube..."
+killall minikube >/dev/null 2>&1 || true
```

Comment:
this kill should happen after we delete minikube so minikube does the right thing.

plus mind that minkube mind not be in the path (the tests download the binary to a different path) so it should not fail if minikube binary is not in the path

Reply:
@medyagh i've amended pr so that first the minikube is searched in each user's path, and if not found, it's downloaded to /tmp and used to execute delete; lastly, killall is executed for each user - just in case that there are any hung processes